Vagrantfile

```diff
@@ -18,9 +18,14 @@ Vagrant.configure(2) do |config|
         echo "America/Detroit" > /etc/timezone
         dpkg-reconfigure -f noninteractive tzdata
 
+        debconf-set-selections <<< 'mysql-server mysql-server/root_password password MySuperPassword'
+        debconf-set-selections <<< 'mysql-server mysql-server/root_password_again password MySuperPassword'
+
```

Comment:
In fact, take a look at this:  https://serversforhackers.com/video/installing-mysql-with-debconf